ansible/roles/magnum/templates/magnum.conf.j2

```diff
@@ -61,7 +61,11 @@ ca_file = {{ openstack_cacert }}
 
 [nova_client]
 region_name = {{ openstack_region_name }}
+{% if magnum_cluster_api_driver_enabled is defined %}
+endpoint_type = publicURL
```

Comment:
What is it that actually needs this? The k8s management cluster? I expect this requirement isn't strict, more about how we deploy it. In which case it might be better as documentation and in our own s-k-c config?

Reply:
@markgoddard  copy and paste from the magnum docs but: 

This is used to generate the application credential config injected into the tenant Kubernetes clusters, such that it is usable from within an OpenStack project, so you can't use the "internal API" end point here.